deploy/sre-prometheus/OWNERS

```diff
@@ -1,4 +1,9 @@
+approvers:
+- rrackow
+= EMEA backup
+- csheremeta
+- NASA backup
+- gsleeman
+- APAC backup
 reviewers:
-- jewzaam
-- mbarnes
-- cblecker
+- @openshift/openshift-team-monitoring
```

Comment:
This will prevent proper reviewers from being assigned.

Reply:
At this point it's a placeholder that needs to be filled with an actual list before merge.